src/main/resources/language/MessagesBundle_en_US.properties

```diff
@@ -113,10 +113,13 @@ home_info = Your homes: {0}
 # {0} = homename
 sethome = You created the home {0}.
 delhome = Your home {0} was deleted.
+jobinfo_blockprice = Mine: {0} {1} {2}
 # {0} = fishing type {1} = price {2} = currency
-jobinfo_fishingprice = Fish {0} {1} {2}
+jobinfo_fishingprice = Fish: {0} {1} {2}
 # {0} = entity {1} = price {2} = currency
-jobinfo_killprice = Kill {0} {1} {2}
+jobinfo_killprice = Kill: {0} {1} {2}
+# {0} = breedable entity {1} = price {2} = currency
+jobinfo_breedingprice = Breed: {0} {1} {2}

```

Comment:
In this specific case it may be possible for us to translate them at least partially. The specific messages needed are _extremely_ close to the existing text for addMob and removeMob. Without knowing the language it's impossible to now the specific context of how they translated "mob" but I think it's not a bad idea for me to go ahead and just copy/paste these two lines to supplement.

Reply:
It occurs to me that the language message code could be rewritten such that if a label entry is missing from a language file it attempts to parse the text from the English file as a default to supplement the line before spitting out the ex. "!jobinfo_breedingprice" empty text. It would probably be more beneficial to end users in the event a translation is missing. Perhaps it should be in combination with the debug text so that people have some way of knowing which label is missing for their language.

English is a good choice as a language to fall back on because people in a lot of other cultures will learn at least basic English as a foreign language.